mods/ra/maps/evacuation/evacuation.lua

```diff
@@ -173,11 +173,11 @@ Tick = function()
 		allies2.MarkFailedObjective(objHoldPosition)
 	end
 
-	if not allies2.IsObjectiveCompleted(objCutSovietPower) and soviets.PowerState ~= "Normal" then
+	if not allies2.IsObjectiveCompleted(objCutSovietPower) and soviets.PowerState ~= "normal" then
```

Comment:
Not sure if this is case-sensitive. But definitely also collateral damage from sed.